File: ing../Spring MVC/Springmvc/src/main/java/com/nhnacademy/springmvc/config/WebConfig.java
package com.nhnacademy.springmvc.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;
import org.springframework.web.servlet.config.annotation.ViewResolverRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
@EnableWebMvc
//@ComponentScan(basePackages = {"com.nhnacademy.springmvc1.**.controller"})
@ComponentScan(basePackageClasses = {com.nhnacademy.springmvc.controller.ControllerBase.class})
public class WebConfig implements WebMvcConfigurer {

    @Override
    public void configureViewResolvers(ViewResolverRegistry registry) {
        registry.jsp("/WEB-INF/view/", ".jsp");
    }

}

File: ing../Spring MVC/Springmvc1/src/main/java/com/nhnacademy/springmvc1/config/WebConfig.kt
package com.nhnacademy.springmvc1.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.EnableWebMvc
import java.lang.Override

@Configuration
@EnableWebMvc
//@ComponentScan(basePackages = {"com.nhnacademy.springmvc1.**.controller"})
@ComponentScan(basePackageClasses = {com.nhnacademy.springmvc1.controller.ControllerBase.class})
public class WebConfig implements WebMvcConfigurer {

    @Override
    public void

}